packages/flutter/lib/fix_data.yaml

```diff
@@ -17,6 +17,273 @@
 
 version: 1
 transforms:
+# Changes made in https://github.com/flutter/flutter/pull/95870/
```

Comment:
These should be as consolidated as possible rather than splitting each change into a separate rule. They should represent  the complete migration paths A, B, C, etc for as many options as there are. `bulkApply: false` should be used since there are options.

Reply:
I have consolidated multiple options (based on different conditions) and divided fixes into different theme_data types
(regular, raw, copyWith) and applied bulkApply to false and removed material expect changes